webview/src/experiments/components/table/styles.module.scss

```diff
@@ -729,15 +730,12 @@ $badge-size: 0.85rem;
 
 .experimentCellText {
   @extend .cellContents;
-  display: flex;
-  flex-flow: row wrap;
```

Comment:
If we're going to keep things stacked on top of each other, I think we need to give the rows more height. Things look pretty crowded vertically when they're wrapped: 

<img width="422" alt="image" src="https://user-images.githubusercontent.com/43496356/233402406-fdcaf515-3607-444e-8762-afa31c6170f4.png">